tests/runtime/builtin

```diff
@@ -63,21 +63,21 @@ TIMEOUT 5
 AFTER ./testprogs/syscall read
 
 NAME sarg
-PROG uprobe:./testprogs/stack_args:too_many_args { printf("SUCCESS %d %d\n", sarg0, sarg1); exit(); }
+PROG uprobe:./testprogs/stack_args:too_many_args { printf("SUCCESS %d %d\n", (int32)sarg0, (int32)sarg1); exit(); }
```

Comment:
This is an interesting example where `%d` was masking the fact that `sarg<N>` reads 8 bytes from the stack (at address `SP + N*8`).

`too_many_args()` is a function that takes `int` arguments, and clang generates the following code:

```
[...]
   0x00000000004005a1 <+17>:    mov    $0x1,%esi
   0x00000000004005a6 <+22>:    mov    $0x2,%edx
   0x00000000004005ab <+27>:    mov    $0x4,%ecx
   0x00000000004005b0 <+32>:    mov    $0x8,%r8d
   0x00000000004005b6 <+38>:    mov    $0x10,%r9d
   0x00000000004005bc <+44>:    movl   $0x20,(%rsp)
   0x00000000004005c3 <+51>:    movl   $0x40,0x8(%rsp)
   0x00000000004005cb <+59>:    movl   $0x80,0x10(%rsp)
   0x00000000004005d3 <+67>:    movl   $0x100,0x18(%rsp)
   0x00000000004005db <+75>:    callq  0x400560 <too_many_args>
```

It only does a 4-byte write (`movl`) to `%rsp`, so the value returned by `sarg0` also includes whatever was already on the stack (the other 4 bytes at `%rsp`).

`%d` truncated previously, but with this change we'd display the full 64-bit value.